godot-macros/src/docs.rs

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) godot-rust; Bromeon and contributors.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+use crate::class::{ConstDefinition, Field, FuncDefinition, SignalDefinition};
+mod md2bb;
+use proc_macro2::{Ident, TokenStream};
+use quote::ToTokens;
+use venial::*;
+
+pub fn define(base: String, description: Vec<Attribute>, members: Vec<Field>) -> TokenStream {
```

Comment:
this name isn't very clear about what this function does

Reply:
it would be nice if you could comment in cases like this what your resolution to the issue was, in this case changing it to `make_definition_docs`. it's definitely not always needed, but here it would be nice.

but it's still not entirely clear to me, is `definition` some established term in godot documentation? i havent seen it used like this before so im unsure if this is a term you came up with or if it's a pre-existing one. If `definition` is the best term to use then maybe you could add some documentation explaining what it means? possibly linking to godot docs if it is a pre-existing term?

Reply:
Im using definition to refer to
```
struct X {}
```